lib/Ix/DBIC/ResultSet.pm

```diff
@@ -295,6 +348,18 @@ sub ix_purge ($self, $ctx, $arg = {}) {
   });
 }
 
+=method ix_create($ctx, $to_create)
+
+This method implements the creation logic for 'Foo/set'; you should not ever
+need to call it independently. It is passed the hashref given to
```

Comment:
"You should not ever need to call it independently". Should we include this for ix_changes and ix_get and such too? Seems weird we list it for some but not others.

Reply:
It's not weird, I think! ix_{create,update,destroy} are called internally by ix_set, and isn't _really_ a top-level JMAPpy method. I'll add some more clarification to the overview to say that you probably won't ever call any of these methods directly.